static/js/composebox_typeahead.js

```diff
@@ -272,12 +272,7 @@ exports.content_typeahead_selected = function (item) {
     } else if (this.completing === 'mention') {
         beginning = (beginning.substring(0, beginning.length - this.token.length-1)
                 + '@**' + item.full_name + '** ');
-
-        // We insert a special `all` item to the autocompleter above
-        // Don't consider it a user mention
-        if (item.email !== 'all' && item.email !== "everyone") {
-            $(document).trigger('usermention_completed.zulip', {mentioned: item});
-        }
+        $(document).trigger('usermention_completed.zulip', {mentioned: item});
```

Comment:
Do you know why that old `if` statement was there?  Would be nice to know if this is somehow introducing another bug.